lib/identity_cache.rb

```diff
@@ -40,6 +40,9 @@ class << self
     mattr_accessor :cache_namespace
     self.cache_namespace = "IDC:#{CACHE_VERSION}:".freeze
 
+    mattr_accessor :fetch_returns_relation
+    self.fetch_returns_relation = false
```

Comment:
Done! Now, set to false. I have added an `allow_fetch_to_return_array` method to go along with this change.